run_all_trials.bash

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+TEAM_NAME=$1
+
+#TODO: get list of trials automatically
```

Comment:
Oh, it was pretty obvious sorry for the question. Let's try a patch to fix all the three comments/problems we have in this file:

```diff
diff --git a/run_all_trials.bash b/run_all_trials.bash
index 3f0822e..7fd7b7c 100755
--- a/run_all_trials.bash
+++ b/run_all_trials.bash
@@ -1,10 +1,24 @@
 #!/usr/bin/env bash
 
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
 TEAM_NAME=$1
+COMP_CONFIGS_DIR=${DIR}/comp_configs/
+
+get_list_of_trials()
+{
+  yaml_files=$(ls ${COMP_CONFIGS_DIR}/*.yaml)
+
+  for f in $(ls ${COMP_CONFIGS_DIR}/*.yaml); do
+    f=${f##*/}
+    f=${f//.yaml}
+    all_names="${all_names} ${f}"
+  done
+
+  echo $all_names
+}
 
-#TODO: get list of trials automatically
-declare -a TRIAL_NAMES=("example_trial1" "example_trial2")
-for TRIAL_NAME in ${TRIAL_NAMES[@]}; do
+for TRIAL_NAME in $(get_list_of_trials); do
   echo "Running trial: ${TRIAL_NAME}"
   #TODO: GET THIS TO RUN THE INDIVIDUAL TRIALS CORRECTLY.
   ./run_trial.bash ${TEAM_NAME} ${TRIAL_NAME}
```